server/lib/config/pbench-server-default.cfg

```diff
@@ -103,6 +103,9 @@ realm = pbench-server
 # Client entity name requesting OIDC to authenticate a user.
 client = pbench-client
 
+# API key expiration duration in minutes, can be overridden in the main config file, defaults to 1 year
+api_key_expiration_duration = 525600
```

Comment:
`api_key_duration_minutes` would be a more useful name!

Reply:
Or, let's just get rid of expiration....  😏